File: gits/My_Homework/leetcode_clone/Sum.cpp
#include <iostream>
#include <fstream>

int sum (int a, int b) {
    return a + b;
}

int main (int a, char* arg[]) {
    std::ofstream file_write("result.txt");
    file_write << sum(std::stoi(arg[1]),std::stoi(arg[2]));
    file_write.close();
    return 0;
}

File: gits/My_Homework/leetcode_clone/Sum.js
function Sum (a,b) {
    return parseInt(a) + parseInt(b)
}
const fs = require('fs');

try {
  fs.writeFileSync('/home/ar/Desktop/leetcode_clone/result.txt', Sum(process.argv[2],process.argv[3]));
  // file written successfully
} catch (err) {
  console.error(err);
}
